community/index.md

```diff
@@ -45,66 +45,51 @@ These forums are covered by the [Scala Code of Conduct](../conduct.html).
 
 Lightbend operates a Discourse forum as well:
 
- * **[discuss.lightbend.com](https://discuss.lightbend.com)**: For discussion of reactive architectures, Akka, Play, Lagom, and related tooling including sbt.
+ * **[discuss.lightbend.com](https://discuss.lightbend.com)**: For discussion of reactive architectures, Akka, Play, and related tooling including sbt.
 
 ## Scala Jobs
 
-Employers and job seekers can find each other in the [scala/job-board Gitter room](https://gitter.im/scala/job-board).
+Employers and job seekers can find each other in the #jobs channel of the [Scala Discord](https://discord.gg/scala).
 
-Job listings are not allowed in our other forums and chat rooms.
+Job postings are not allowed in our other forums and chat rooms.
 
 ## Chat Rooms
 
-Our main chat platform is [Gitter](https://gitter.im).  See next section for other platforms.
+Our main chat platform is Discord, and the main Scala server is:
 
-The main Gitter room for Scala is:
+* **[Scala](https://discord.com/invite/scala)**
+  * the #scala-users channel is especially beginner-friendly
+  * the #scala-contributors channel is about moving Scala forward
+  * the #jobs channel is the only place we allow job postings
+  * ask on #admin if you have questions or suggestions about the server itself
+  * there are many other channels, including #spark, #scala-js, and #scala-native
 
-* **[scala/scala](https://gitter.im/scala/scala)**: Questions, general discussion, etc. Beginner-friendly.
+The server is covered by the [Scala Code of Conduct](../conduct.html).
 
-Other, more specialized rooms include:
+Scala-oriented Discord servers operated by the community include:
 
-* **[scala/center](https://gitter.im/scala/center)**: for discussions about Scala community governance, processes, the Scala Platform, and projects going on at the Scala Center.
-* **[scala/contributors](https://gitter.im/scala/contributors)**: for contributors to discuss work on changes to Scala.
-* **[scala/moocs](https://gitter.im/scala/moocs)**: for talking about the Scala Center's online courses
```

Comment:
Is there a replacement for this room? I am not very familiar with Discord, but I couldn’t find an equivalent channel. Should we redirect users of https://gitter.im/scala/moocs to the scala-users Discord channel?

Reply:
Traffic in the old room was so sparse that I don't think it matters.

Rather than take the existence of the old room as a precedent and starting point, I would suggest starting over and asking: doyou want a room, is there a need for a room, will somebody involved with the MOOCs monitor it?

The old room got very little traffic, so I personally don't see a need for a room _unless_ you're prepared to publicize it and make efforts to send people there.